lib/cocoapods/sandbox/path_list.rb

```diff
@@ -57,6 +57,11 @@ def read_file_system
         root_length = root.cleanpath.to_s.length + File::SEPARATOR.length
         Find.find(root.to_s) do |f|
           directory = File.directory?(f)
+          path = Pathname.new(f)
+          # File.symlink? does not detect folder symlinks
+          if File.symlink?(f) || (path.realpath != path)
+            UI.warn "Symlink `#{f}` found while traversing `#{root}`. Symlinks are not traversed."
```

Comment:
it might help to say what it symlinks to? (and maybe just keep a dictionary of what's found and do one big warning at the end?)

I'm also just generally worried about the performance of this as a whole, and wonder whether it might be better delegated to the linter?

Reply:
Yeah, this also appears to be triggering a lot of warnings for vendored frameworks so I don't think this is the right way to go.